instrumentation-core/spring/build.gradle

```diff
@@ -0,0 +1,40 @@
+plugins {
+	id 'org.springframework.boot' version '2.3.0.RELEASE'
+	id 'io.spring.dependency-management' version '1.0.9.RELEASE'
+	id 'java'
+}
+
+group = 'io.opentelemetry.instrumentation'
+version = '0.0.1-SNAPSHOT'
+
+apply from: "$rootDir/gradle/java.gradle"
+
+sourceCompatibility = '8'
+
+configurations {
+	compileOnly {
+		extendsFrom annotationProcessor
+	}
+}
+
```

Comment:
is this for lombok? maybe u don't need this anymore also with "java.gradle" above

Reply:
Yes, it's just for lombok. I'll remove it now.